src/parser.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+
+def parse():
+    args = argparse.ArgumentParser(
+        prog="Wilson's comic collection",
+        description="""
+            Loops through a directory of files extracting necessary 
+            metadata from file names into a csv to be used for 
+            cataloging Jeff's catalog collection. If you are trying to 
+            use this for something other than make spreadsheets for 
+            Jeff's comic collection - may god have mercy on your soul.
+            """,
+        add_help=True,
+    )
+
+    args.add_argument(
+        "--directory",
+        "-d",
+        type=str,
+        default="./Photos/",
```

Comment:
Note if the `--directory` flag isn't passed to the program we will use the top photos directory. You could argue that this should also be some type of global variable but 🤷